src/components/List/Cells/DeadlineCell.tsx

```diff
@@ -1,8 +1,23 @@
 import React from 'react'
 import { CellProps } from '../../../utils/types'
+import { DateTime } from 'luxon'
+
+// Show date-time in a different colour (red) if deadline is less than this many
+// days away:
+const ALERT_FORMAT_DAYS = 2
 
 const DeadlineCell: React.FC<CellProps> = ({ application }) => {
-  // TODO
-  return null
+  if (!application.applicantDeadline) return null
+
+  const deadline = DateTime.fromISO(application.applicantDeadline)
+  const textColorClass =
+    deadline.diff(DateTime.now(), 'days').days < ALERT_FORMAT_DAYS ? 'alert-colour' : ''
+  return (
+    <p className={`slightly-smaller-text ${textColorClass}`}>
+      {deadline.toLocaleString()}
+      <br />
+      {deadline.toLocaleString(DateTime.TIME_SIMPLE)}
```

Comment:
hmm, wondering if we need this, since scheduler runs every hour (might have expired but not quite set to expired yet), i guess technically it should be rounded to next hour. All good to keep i guess